src/localize/languages/pt-pt.json

```diff
@@ -0,0 +1,467 @@
+    "view": "Visualizar",
+    "view_secondary": "O que o cartão deve mostrar e como mostrá-lo"
+  },
+  "error": {
+    "could_not_render_elements": "Não foi possível renderizar os elementos da imagem",
+    "could_not_resolve": "Não foi possível resolver o URL de mídia",
+    "diagnostics": "Diagnósticos do cartão. Revise as informações confidenciais antes de compartilhar",
+    "download_no_media": "Nenhuma mídia para download",
+    "download_sign_failed": "Não foi possível assinar o URL de mídia para download",
+    "duplicate_camera_id": "Duplique o ID da câmera Frigate para a câmera a seguir, use o parâmetro 'id' para identificar exclusivamente as câmeras",
+    "empty_response": "Sem resposta do Home Assistant para a solicitação",
+    "failed_response": "Falha ao receber resposta do Home Assistant para solicitação",
+    "failed_retain": "Não foi possível reter o evento",
+    "failed_sign": "Não foi possível assinar a URL do Home Assistant",
+    "image_load_error": "A imagem não pôde ser carregada",
+    "invalid_configuration": "Configuração inválida",
+    "invalid_configuration_no_hint": "Nenhuma dica de local disponível (tipo incorreto ou ausente?)",
+    "invalid_elements_config": "Configuração de elementos de imagem inválida",
+    "invalid_response": "Resposta inválida recebida do Home Assistant para a solicitação",
+    "jsmpeg_no_player": "Não foi possível iniciar o player JSMPEG",
+    "live_camera_no_endpoint": "sem câmera ao vivo",
+    "live_camera_not_found": "câmera ao vivo não encontrada",
+    "live_camera_unavailable": "câmera ao vivo indisponivel",
+    "no_camera_engine": "Não existe câmera",
+    "no_camera_entity": "Não existe uma entidade câmera  ",
+    "no_camera_entity_for_triggers": "Não existe camera para a acção",
+    "no_camera_id": "Não foi possível determinar o ID da câmera para a câmera a seguir, pode ser necessário definir o parâmetro 'id' manualmente",
+    "no_camera_name": "Não foi possível determinar o nome da câmera da Frigate, especifique 'camera_entity' ou 'camera_name' para a câmera a seguir",
+    "no_live_camera": "O parâmetro camera_entity deve ser definido e válido para este serviço ativo",
+    "no_visible_cameras": "",
+    "reconnecting": "Reconectando",
+    "timeline_no_cameras": "Nenhuma câmera do Frigate para mostrar na linha do tempo",
+    "troubleshooting": "Verifique a solução de problemas",
+    "unknown": "Erro desconhecido",
+    "upgrade_available": "Uma atualização automatizada da configuração do cartão está disponível, visite o editor visual do cartão",
+    "webrtc_card_reported_error": "O cartão WebRTC relatou um erro",
+    "webrtc_card_waiting": "Aguardar o cartão WebRTC carregar ..."
+  },
+  "event": {
+    "camera": "camera",
+    "duration": "Duração",
+    "in_progress": "Em andamento",
+    "score": "Pontuação",
+    "seek": "Procurar",
+    "start": "Início",
+    "what": "o quê",
+    "where": "onde"
+  },
+  "media_filter": {
+    "all": "todos",
```

Comment:
For many of the entries here I notice the first letter is not capitalized, where as it is in `en` and `pt-BR`. I am not qualified to assess if this is correct or not :-) I don't think I can think of a single example where the strings here are mixed into an existing string, i.e. are always the start of a sentence or title, which means they are always capitalized.

Just sanity checking if the same is true in `pt-pt`!